vscode-antimony/src/server/stibium/stibium/analysis.py

```diff
@@ -225,6 +230,8 @@ def check_parse_tree(self, root, scope):
                     self.process_is_assignment(node, scope)
                 elif type(node.get_stmt()) == Assignment:

```

Comment:
why not "isinstance(node.get_stmt(), Assignment)"? Also, why don't you do "stmt = node.get_stmt()" at the top of the if block and then "isinstance(stmt, Assignment)" here and "isinstance(stmt, Event)" for the next elif?

Reply:
This would be an issue to fix rather than fix it in this PR.